### PR TITLE
Add file size limit to 25MB

### PR DESCRIPTION
**Pull request by kahlouie**
## Description

The copy has been changed to: `Please keep each file under 25MB.`
Too limit in-memory issues, we're adding a limit to the size of file that can be uploaded to 25MB, which is already fairly big. Notably, I thought about adding this directly to the uploader component, but on the SM weight ticket pages, it would appear three times if I did that and the layout looked wonky.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```
Try uploading a document greater than 25MB from the document uploader screen.

```sh
make server_run
make client_run
```
As a brand new SM, create your profile and attempt to upload orders greater than 25MB.
Try editing orders upload after that.

Then as `ppm@requestingpay.ment`, go through the flow. At each upload point, try to upload a file greater than 25MB.

## Code Review Verification Steps

* [x] Request review from a member of a different team.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* https://www.pivotaltracker.com/story/show/167658360

## Screenshots
The copy on the screens should now all be `Please keep each file under 25MB.` instead of `We do not accept files larger than 25 MB.`
Expenses page on PPM Closeout:
Screen Shot 2019-08-05 at 1 59 57 PM
Weight ticket page on PPM Closeout:
Screen Shot 2019-08-05 at 1 56 02 PM
Screen Shot 2019-08-05 at 1 55 51 PM
Orders upload with a small enough file:
Screen Shot 2019-08-05 at 1 42 09 PM
Orders upload with too big of a file:
Screen Shot 2019-08-05 at 1 41 56 PM
Office document uploader:
Screen Shot 2019-08-05 at 1 33 53 PM

### CODE REVIEW

File: pkg/uploader/uploader.go [hunk 1]
```diff
@@ -17,6 +17,11 @@ import (
 // ErrZeroLengthFile represents an error caused by a file with no content
 var ErrZeroLengthFile = errors.New("File has length of 0")
 
+const fileSizeLimit = 25000000
+
+// ErrTooLarge represents an error caused by a file larger than 25MB
+var ErrTooLarge = errors.New("File is too large. We do not accept files larger than 25 MB")
```
Comment (mkrump) [inline pkg/uploader/uploader.go]: 👍

File: pkg/uploader/uploader.go [hunk 2]
```diff
@@ -55,6 +60,9 @@ func (u *Uploader) CreateUploadForDocument(documentID *uuid.UUID, userID uuid.UU
 	if info.Size() == 0 {
 		return nil, responseVErrors, ErrZeroLengthFile
 	}
+	if info.Size() > fileSizeLimit {
```
Comment (mkrump) [inline pkg/uploader/uploader.go]: could probably add a test similar to `TestUploadFromLocalFileZeroLength` in `uploader_test.go`

File: src/scenes/Moves/Ppm/WeightTicket.jsx [hunk 3]
```diff
@@ -271,6 +271,7 @@ class WeightTicket extends Component {
                       <p className="normalize-margins" style={{ marginTop: '1em' }}>
                         Proof of ownership (ex. registration, bill of sale)
                       </p>
+                      <p>Please keep each file under 25MB.</p>
```
Comment (mkrump) [inline src/scenes/Moves/Ppm/WeightTicket.jsx]: If we're using the same design approved language over and over, might make sense to pull out into a shared const file so others can find easily and make it a little easier to change in future if need to.

Comment (kahlouie) [reply src/scenes/Moves/Ppm/WeightTicket.jsx]: Great suggestion.

**Comment by kahlouie** [comment]
@mkrump re: 
> Notably, I thought about adding this directly to the uploader component, but on the SM weight ticket pages, it would appear three times if I did that and the layout looked wonky.
I tried to add the copy to the uploader component as a p tag text. The problem was that on the `src/scenes/Moves/Ppm/WeightTicket.jsx` the uploader can appear 3 times. Once for a trailer, and two times for full and empty weight tickets. That changed the layout on the uploader. Adding the text directly in the p tag gave me more control over where the text appeared.

**Pull request merged**